src/Log/Engine/SyslogLog.php

```diff
@@ -106,10 +123,9 @@ public function log($level, $message, array $context = []): void
             $priority = $this->_levelMap[$level];
         }
 
-        $messages = explode("\n", $this->_format($message, $context));
-        foreach ($messages as $message) {
-            $message = sprintf($this->_config['format'], $level, $message);
-            $this->_write($priority, $message);
+        $lines = explode("\n", $this->_format($message, $context));
+        foreach ($lines as $line) {
+            $this->_write($priority, $this->formatter->format($level, $line, $context));
```

Comment:
Should each line be formatted? We're mostly force-wrapping the long line to work around protocol deficiencies.

Reply:
I don't use Syslog so I can't say. I was only trying to re-create the special formatting that existed. It was adding the level as a prefix to each line.

Reply:
I guess this will work well with JSON formatting. For example logging a long string into a JSON format will result in multiple messages (one per line) and each line is JSON formatted with additional context.